### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rsappia**
Just added some specific content related to the raspberry pi zero setup :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,4 +2,117 @@
...
+`cd /home/pi/fork/retro-ipod-spotify-client/frontend/`
+
+`sudo -H -u pi python3 spotifypod.py &`
+
+`sudo /home/pi/fork/retro-ipod-spotify-client/clickwheel/click &`
+
+
+_Make sure that the paths are ok with your setup!!_
+
+in ` sudo nano /etc/xdg/openbox/environment` I set all the variables needed to run spotifypod.py ( SPOTIPY_CLIENT_ID, SPOTIPY_CLIENT_SECRET,SPOTIPY_REDIRECT_URI)
```
Comment (dupontgu) [inline README.md:77]: Can you remove the "I" here? Now that folks are actually adding to/using this repo, I feel like we should move away from using identifiers. I will do the same for my original contributions as well.

File: README.md [hunk 2]
```diff
@@ -2,4 +2,117 @@
...
+
+**Synchronizing Spotify data!**
+Last but not least, if you want to make sure all your playlists artists, etc are synchronized every time you turn on your Spotypod, you can simply modify the script view_model.py with the following at line 16:
+
+`#spotify_manager.refresh_devices()`
+
+`spotify_manager.refresh_data()`
+
+
+instead of calling refresh_device, you can execute refresh_data. This will sync all your data and then will eceute refresh.devices. This will make the boot up way slower! but I will synchronize every single time you switch on :).
```
Comment (dupontgu) [inline README.md:95]: Same here, no "I" please.

**Review by dupontgu — CHANGES_REQUESTED** [review]
Hi, thank you for writing this up! I made some very small requests.

**Comment by dupontgu** [comment]
And actually - Sorry for the trouble, but I had approved a PR with changes to this file a few weeks ago that I realize was never merged! 
https://github.com/dupontgu/retro-ipod-spotify-client/pull/2/files

I think your instructions will still work well alongside what this author added, but you'll have to resolve conflicts. sorry about that!

File: README.md (hunk 1 above)
Comment (rsappia) [reply README.md]: done!

File: README.md (hunk 2 above)
Comment (rsappia) [reply README.md]: ups... that was a typo. Done!

**Review by dupontgu — APPROVED** [review]
Awesome, thanks. I will merge.

**Pull request merged**